src/System.Runtime.Serialization.Json/tests/Performance/DcjsDeserializationTests.cs

```diff
@@ -3,139 +3,35 @@
-
-            using (var stream = new MemoryStream())
-            {
-                serializer.WriteObject(stream, obj);
-                foreach (var iteration in Benchmark.Iterations)
-                {
-                    using (iteration.StartMeasurement())
-                    {
-                        for (int i = 0; i < iterations; i++)
-                        {
-                            stream.Position = 0;
-                            serializer.ReadObject(stream);
-                        }
-                    }
-                }
-            }
-        }
-
-        public static byte[] CreateByteArray(int size)
-        {
-            byte[] obj = new byte[size];
-            for (int i = 0; i < obj.Length; ++i)
-            {
-                obj[i] = (byte)(i % 256);
-            }
-            return obj;
-        }
-
-        [Benchmark]
-        public void DeserializeByteArrayOf1K()
-        {
-            byte[] byteArrayOf1K = CreateByteArray(1024);
-            RunDcjsDeserializationTest(byteArrayOf1K, 10000);
-        }
-
-        [Benchmark]
-        public void DeserializeByteArrayOf1M()
-        {
-            byte[] byteArrayOf1M = CreateByteArray(1024 * 1024);
-            RunDcjsDeserializationTest(byteArrayOf1M, 10);
-        }
-
-        [Benchmark]
-        public void DeserializeStringOf128Byte()
-        {
-            string stringOf128Bytes = new string('a', 128);
-            RunDcjsDeserializationTest(stringOf128Bytes, 10000);
-        }
-
-        [Benchmark]
-        public void DeserializeStringOf1024Byte()
-        {
-            string stringOf1024Bytes = new string('k', 1024);
-            RunDcjsDeserializationTest(stringOf1024Bytes, 10000);
-        }
-
-        private static List<int> CreateListOfInt(int count)
-        {
-            return Enumerable.Range(0, count).ToList();
-        }
-
-        [Benchmark]
-        public void DeserializeListOfInt128()
-        {
-            List<int> listOfInt = CreateListOfInt(128);
-            RunDcjsDeserializationTest(listOfInt, 1000);
-        }
-
-        [Benchmark]
-        public void DeserializeListOfInt1K()
-        {
-            List<int> listOfInt = CreateListOfInt(1024);
-            RunDcjsDeserializationTest(listOfInt, 1000);
-        }
-
-        [Benchmark]
-        public void DeserializeListOfInt1M()
-        {
-            List<int> listOfInt = CreateListOfInt(1024 * 1024);
-            RunDcjsDeserializationTest(listOfInt, 1);
-        }
-
-        private static Dictionary<int, string> CreateDictionaryOfIntString(int count)
-        {
-            Dictionary<int, string> dictOfIntString = new Dictionary<int, string>();
-            for (int i = 0; i < count; ++i)
-            {
-                dictOfIntString[i] = i.ToString();
-            }
-
-            return dictOfIntString;
+            return PerformanceTestCommon.PerformanceMemberData(SerializerType.DataContractJsonSerializer);
         }
 
         [Benchmark]
```

Comment:
@brianrob Yes I've followed the guideline for perf tests and we do use Benchmark.Ierations and StartMeasurement in the main method to run serialization perf test at: https://github.com/dotnet/corefx/pull/7119/files#diff-392ee56ad323b27581f82a43af91af89R160